Source/Core/VideoCommon/NetplayUI.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "VideoCommon/NetplayUI.h"
+
+#include <utility>
+#include <vector>
+
+#include "imgui.h"
+
+constexpr size_t MAX_BACKLOG_SIZE = 100;
+
+static char message_buf[160] = {};
```

Comment:
NetplayUI should really be a class to hold all of this state, instead of making it permanently take up memory, even if the UI isn't even being used.